docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -333,6 +333,24 @@ nQIDAQAB
 -----END PUBLIC KEY-----
 ----
 
+=== Generating Keys
```

Comment:
@nderwin Thanks for this update, IMHO it would be better to keep it not as one of the main subsections, but as a `Note`, `Generating Keys with openssl`, as generating the keys openssl is not necessarily what the users will generally do and have this note immediately under the private key pem example, instead of injecting `(or use the privateKey.pem file from <<Generating Keys>>)` (otherwise we;ll also need to do the same for the public key).
So have this note and suggest the end - that `you can use the generated pair of keys instead of the keys used by this demo` or something like that...

Does it sound reasonable ?

Reply:
Sure!  I'll make those changes and update the PR shortly.